.github/workflows/push_and_pull_crowdin_translations.yml

```diff
@@ -0,0 +1,63 @@
+name: Push and pull Crowdin translations
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  push_and_pull_crowdin_translations:
+    if: github.repository == 'binary-com/deriv-app'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup node
+        uses: actions/setup-node@v1
```

Comment:
in `.github/workflows/generate_and_push_deriv_api_types.yml` file `setup-node@v2` was used. Would you please explain why a previous version was selected for this workflow?

Reply:
Classic copy pasterino here, let me update this. 😨